client/src/components/ExpandedRowDataView.tsx

```diff
@@ -58,8 +64,10 @@ const ExpandedRowDataView = ({data, onError, onUpdate, user}: DataViewProps) =>
                 <div className="left">
                     {showClaim && (<ActionButton label="Claim" reviewId={data.id}/>)}
                     {showChange && (<ActionButton label="Request Change" reviewId={data.id}/>)}
+                    {showRequestReview && (<ActionButton label="Request Review" reviewId={data.id}/>)}
```

Comment:
Not a blocker, but it might be good to use ternaries here and below

```suggestion
                    {showRequestReview ? <ActionButton label="Request Review" reviewId={data.id}/> : null}
```